### PR TITLE
See if types always match

### PR DESCRIPTION
**Pull request by charleskawczynski**
Running a smoke test with this to see if anything fails. This is a follow up to #336.

### CODE REVIEW

**Comment by charleskawczynski** [comment]
bors try

File: src/Operators/finitedifference.jl [hunk 1]
```diff
@@ -2148,6 +2148,12 @@ function apply_stencil!(field_out, bc)
     lw = left_interor_window_idx(bc, space, lbw)
     ri = right_idx(space)
     rw = right_interor_window_idx(bc, space, rbw)
+    if !(typeof(ri) == typeof(rw))
+        error("oops")
```
Comment (valeriabarra) [inline src/Operators/finitedifference.jl:2152]: I am always in favor of more informative error messages :)

Comment (charleskawczynski) [reply src/Operators/finitedifference.jl:2152]: Haha, yeah, this is just to test if this condition is generally true or not. I'll be removing it :)

Comment (jakebolewski) [reply src/Operators/finitedifference.jl:2152]: this will always be true because the space is checked before the broadcasted expression is materialized

**Pull request closed without merging**